pages/[chain]/collection/[contract].tsx

```diff
@@ -543,16 +546,17 @@ const CollectionPage: NextPage<Props> = ({ id, ssr }) => {
                       openState={isMintRoute ? mintOpenState : undefined}
                       buttonChildren={
                         <Flex
-                          css={{ gap: '$2', px: '$4' }}
+                          css={{ gap: '$2', px: '$2' }}
                           align="center"
                           justify="center"
                         >
                           {isSmallDevice && (
                             <FontAwesomeIcon icon={faSeedling} />
                           )}
+                          {isSmallDevice && (                         
                           <Text style="h6" as="h6" css={{ color: '$bg' }}>
```

Comment:
So will minting not be supported on mobile?

Reply:
<img width="356" alt="Screenshot 2023-11-13 at 10 08 59 AM" src="https://github.com/reservoirprotocol/marketplace-v2/assets/97770763/d4d42004-5f68-4110-b8b3-1ea06cc09346">

Im not sure I understand what you mean. The mint button is the little seedling now.

Reply:
oh right, forgot about this. We're just removing the copy. Makes sense